<launch>

  <arg name="debug" default="false"/>
  <arg if="$(arg debug)" name="launch_prefix" value="xterm -rv -e gdb -ex run -args"/>
  <arg unless="$(arg debug)" name="launch_prefix" value=""/>

  <node name="ARProsilicaSinglePattern" args="ARProsilicaSinglePattern" pkg="ar_pose" type="ar_single" respawn="false" output="screen" launch-prefix="$(arg launch_prefix)">
    <rosparam command="load" file="$(find ar_pose)/config/ar_single_marker.yaml"/>
    <param name="camera_frame" type="string" value="/PROSILICA"/>    
    <param name="marker_frame" type="string" value="ARProsilicaMarker"/>
    <param name="camera_image_topic" type="string" value="/Prosilica/image_rect_color"/>
    <param name="camera_info_topic" type="string" value="/Prosilica/camera_info"/>
    <param name="marker_red" type="double" value="0.0"/>
    <param name="marker_green" type="double" value="0.0"/>
    <param name="marker_blue" type="double" value="1.0"/>
  </node>
</launch>
